script.js

```diff
@@ -1,22 +1,118 @@
-const courseContainer = document.querySelector(".courses-container");
 const searchButton = document.querySelector(".search-btn");
+const courseList = document.querySelectorAll(".course-bar li");
+const sectionContainer = document.querySelector(".course-container");
 let courses = [];
+[...courseList].forEach(function(link) {
+
+    link.addEventListener('click', function(e) {
+        e.preventDefault();
+        fetchFromLink(link);
+        document.querySelector(".search-input").value = "";
+    });
+
+});
+activate(courseList[0]);
+fetchFromLink(courseList[0]);
+
+function activate(li) {
+    [...courseList].forEach(function(link) {
+        link.classList.remove("course-bar-item-active");
+    });
+    li.classList.add("course-bar-item-active");
+}
+
+function fetchFromLink(link) {
+    fetch(link.getAttribute("href"))
+        .then((response) => response.json())
+        .then(function(container) {
+            courses = container['courses'];
+            return container;
+        }).then((container) => {
+            updateContent(container);
+            return link;
+        })
+        .then((link) => {
+            activate(link);
+        });
+}
+
+
+function updateContent(container) {
+    sectionContainer.innerHTML = "";
+    sectionTitle = document.createElement("h3");
+    sectionTitle.classList.add("course-item", "section-title");
+    sectionTitle.innerHTML = container['sectionTitle'];
+    sectionContainer.appendChild(sectionTitle);
+    courseDesc = document.createElement("p");
+    courseDesc.classList.add("course-item", "paragraph", "section-desc");
+    courseDesc.innerHTML = container['courseDesc'];
+    sectionContainer.appendChild(courseDesc);
+    exploreButton = document.createElement("button");
+    exploreButton.classList.add("course-item", "btn-outline", "explore-button");
+    exploreButton.innerHTML = "Explore " + container['courseName'];
+    sectionContainer.appendChild(exploreButton);
+    updateTable();
+}
 searchButton.addEventListener("click", function(e) {
     e.preventDefault();
+    sectionContainer.removeChild(document.querySelector(".swiper"));
     updateTable();
+
 });
 const searchText = document.querySelector(".search-input");
-searchText.addEventListener("keyup", function(e) {
-    updateTable();
-});
 
 function updateTable() {
     let here = [];
     here = filter(searchText.value);
-    courseContainer.innerHTML = "";
-    for (let i = 0; i < courses.length; i++) {
-        render(here[i]);
+    coursesContainer = document.createElement("div");
+    coursesContainer.classList.add("swiper-wrapper");
+    // console.log(here);
+    for (let i = 0; i < here.length; i++) {
+
+        render(coursesContainer, here[i]);
     }
+    swip = buildSwiper();
+    swip.appendChild(coursesContainer);
+    var swiper = new Swiper(swip, {
```

Comment:
I know `Swiper` works greet and it is easier in imp but the target for this task is to know how to use Bootstrap so you should use carousel component from Bootstrap

Reply:
i actually used bootstrap but not in that thing , i haven't noticed that we should use bootstrap specifically here , so  i used 
swiper which i saw easier and do the same thing

Reply:
okay but the instructor wants you to use the bootstrap carousel, so please if you have time update it